README.md

```diff
@@ -707,6 +707,7 @@ Table of Contents
   * [lean20.com](https://lean20.com/product/status/) - Public status pages for incident reporting. 100% free.
   * [instatus.com](https://instatus.com) - Get a beautiful status page in 10 seconds. Free forever with unlimited subs and unlimited teams.
   * [Squadcast.com](https://squadcast.com) - Squadcast is an end-to-end incident management software that's designed to help you promote SRE best practices. Free forever plan available for upto 10 users.
+  * [RoboMiri.com](https://robomiri.com/) - RoboMiri is a stable uptime monitor that offer a wide range of monitors: cronjob, keyword, website, port, ping. 25 uptime checks with a 3 minutes interval checks for free. Alerts via Phone Call, SMS, Email and Web Hook.
```

Comment:
Is this actually a functional service? none of the links below do anything or take you anywhere. I dont think I'll be adding such a broken service.

<img width="158" alt="Screenshot 2022-03-03 at 13 26 25" src="https://user-images.githubusercontent.com/82342/156564478-ce96ec52-49c9-4124-8f29-9b03eb8c975f.png">

Reply:
yes, will mark this as draft atm then